src/components/Pages/HomeSections/whyUs/whyUsCardsData.tsx

```diff
@@ -0,0 +1,71 @@
+import FreediFirstSliderIcon1 from "@/components/icons/FreediFirstSliderIcon1";
+import FreediFirstSliderIcon2 from "@/components/icons/FreediFirstSliderIcon2";
+import FreediFirstSliderIcon3 from "@/components/icons/FreediFirstSliderIcon3";
+import FreediSecondSliderIcon1 from "@/components/icons/FreediSecondSliderIcon1";
+import FreediSecondSliderIcon2 from "@/components/icons/FreediSecondSliderIcon2";
+import FreediSecondSliderIcon3 from "@/components/icons/FreediSecondSliderIcon3";
+import FreediThirdSliderIcon1 from "@/components/icons/FreediThirdSliderIcon1";
+import FreediThirdSliderIcon2 from "@/components/icons/FreediThirdSliderIcon2";
+import FreediThirdSliderIcon3 from "@/components/icons/FreediThirdSliderIcon3";
+import FreediThirdSliderIcon4 from "@/components/icons/FreediThirdSliderIcon4";
+
+import { CardModel } from "./whyUsModel";
+
+export const firstConnectionCards:CardModel[] = [
+    {
+        title: "Consensus reaching technology",
+        description: "Consensus-reaching technology is an exciting field that aims to help groups of people agree more effectively and fairly through innovative computational methods.",
+        img: <FreediFirstSliderIcon1/>,
+    },
+    {
+        title: "Everybody can Paricipate",
```

Comment:
Typo: Paricipate not correct

Reply:
done, I didn't notice, I just copied it from the figma.